src/lib.rs

```diff
@@ -22,18 +22,16 @@ fn fastuuid(_py: Python, m: &PyModule) -> PyResult<()> {
     #[pymethods]
     impl UUID {
         #[new]
-        #[allow(clippy::new_ret_no_self)]
         #[allow(clippy::too_many_arguments)]
         fn new(
-            obj: &PyRawObject,
             hex: Option<&str>,
             bytes: Option<Py<PyBytes>>,
             bytes_le: Option<Py<PyBytes>>,
             fields: Option<Py<PyTuple>>,
```

Comment:
Well, I've managed to pinpoint the bug which is causing the proc macro to throw away spans - something about `Option<Py<T>>` as argument type. Very weird, I'll debug that another time.

For now, you can fix the compile errors which should making upgrading easier by migrating to `Option<&PyTuple>` etc instead:

```suggestion
            bytes: Option<&PyBytes>,
            bytes_le: Option<&PyBytes>,
            fields: Option<&PyTuple>,
```